src/annotator/guest.js

```diff
@@ -355,9 +341,8 @@ export default class Guest extends Delegator {
 
     removeAllHighlights(this.element);
 
-    for (let name of Object.keys(this.plugins)) {
-      this.plugins[name].destroy();
-    }
+    this.documentMeta.destroy();
```

Comment:
The `DocumentMeta` class doesn't actually have a `destroy` implementation, but since it currently inherits from `Delegator`, invoking `destroy` is the proper thing to do. In future we can change `DocumentMeta` to no longer inherit from `Delegator` and then this can be removed.